v1_19/src/main/java/com/denizenscript/denizen/nms/v1_19/impl/BiomeNMSImpl.java

```diff
@@ -61,10 +58,21 @@ public float getHumidity() {
     }
 
     @Override
-    public float getTemperature() {
+    public float getBaseTemperature() {
         return biomeHolder.value().getBaseTemperature();
     }
 
+    @Override
+    public float getTemperatureAt(Location location) {
+        //noinspection deprecation

```

Comment:
huh?

Reply:
`Biome#getTemperature(BlockPos)` is marked as deprecated, presumably because Mojang wanted to mark it as an internal thing (as it's still used in code) - was just suppressing the irrelevant warning, but can remove that if you'd like.

Reply:
Yeah that suppression method isn't standard, that's... some form of IDE specific thing that shouldn't be in the Denizen codebase